dateparser_cli/cli.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+from .fasttext_manager import fasttext_downloader
+from .utils import clear_cache
+from .exceptions import CommandNotFound
+
+
+_cli_functions_map = {
```

Comment:
Why do we need this map? I think we only refer to it with static keys, and could instead call the functions directly

Reply:
done 👍🏻